lib/cisco_node_utils/bgp_af.rb

```diff
@@ -533,7 +591,7 @@ def redistribute=(should)
       return if delta_hash.values.flatten.empty?
       [:add, :remove].each do |action|
         Cisco::Logger.debug("redistribute delta #{@get_args}\n #{action}: " \
-                            "#{delta_hash[action]}")
+                          "#{delta_hash[action]}")
```

Comment:
ditto comment about indentation. Yet rubocop is happy with it, it looks like - I'm confused. :-)